src/Fabulous.Avalonia.ColorPicker/CHANGELOG.md

```diff
@@ -7,7 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
```

Comment:
It's recommended to put `_No unreleased changes_` under Unreleased in case there is nothing.
Otherwise the Changelog reader plugin might crash.